src/isolate.rs

```diff
@@ -330,4 +372,49 @@ mod tests {
     let op = Box::new(futures::future::ok(control));
     (true, op)
   }
+
+  #[test]
+  fn test_metrics() {
+    let argv = vec![String::from("./deno"), String::from("hello.js")];
+    let mut isolate = Isolate::new(argv, metrics_dispatch);
+    tokio_util::init(|| {
+      // verify that metrics have been properly initialized
+      {
+        let g = isolate.state.metrics.lock().unwrap();
+        let metrics = g.deref();
```

Comment:
Similar comment here regarding `Deref`, normally the implicit deref coercion takes care of things.